bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import subprocess
+from pathlib import Path
+
+def uniffi_gen(executable: str, *args):
+    args = list(str(a) for a in args)
+    uniffi = [executable]
+    uniffi.extend(args)
+    print(f'Running {" ".join(uniffi)}')
+    subprocess.run(uniffi, check=True)
+
+def uniffi_gen_go(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-go", udl, "-o", output_dir)
+
+
+def uniffi_gen_cs(udl: Path, output_dir: Path, config_file: Path):
+    uniffi_gen("uniffi-bindgen-cs", udl, "-o", output_dir, "-c", config_file)
+
+
+def uniffi_gen_swift(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "swift", "-o", output_dir)
+
+
+def uniffi_gen_kotlin(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "kotlin", "-o", output_dir)
+
+def uniffi_gen_python(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "python", "-o", output_dir)
+
+def uniffi_gen_cpp(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-cpp", udl, "-o", output_dir)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Generate UniFFI bindings from the UDL file", exit_on_error=True)
+    parser.add_argument("-d", "--directory", help="Output directory. Default set to './dist/ffi'")
+    parser.add_argument("-u", "--udl", help="UDL file")
+    parser.add_argument("-c", "--config", help="Uniffi config file")
+    parser.add_argument("-a", "--all", help="Generate all bindings", action='store_true')
+    parser.add_argument("--go", help="Generate go bindings", action='store_true')
+    parser.add_argument("--csharp", help="Generate charp bindings", action='store_true')
+    parser.add_argument("--swift", help="Generate swift bindings", action='store_true')
+    parser.add_argument("--kotlin", help="Generate kotlin bindings", action='store_true')
+    parser.add_argument("--python", help="Generate python bindings", action='store_true')
+    parser.add_argument("--cpp", help="Generate cpp bindings", action='store_true')
+    args = parser.parse_args()
+
+    output_dir = Path(args.directory)
```

Comment:
This will fail if nothing is passed as arguments. it would be good to have a guard against it, Running just the script produced me:
```
  File "/home/lukaspukenis/Development/libdrop/./bindings/generate_bindings.py", line 44, in <module>
    output_dir = Path(args.directory)
  File "/usr/lib/python3.10/pathlib.py", line 960, in __new__
    self = cls._from_parts(args)
  File "/usr/lib/python3.10/pathlib.py", line 594, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/usr/lib/python3.10/pathlib.py", line 578, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Reply:
I made these output directory and udl file as required. Should be fine now.